stories/core/Typography/Anchor.stories.tsx

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { Anchor } from '../../../src/core';
+
+export default {
+  title: 'Typography/Anchor',
+  component: Anchor,
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+  },
+  argTypes: {},
+} as Meta;
+
+export const Basic: Story<React.ComponentPropsWithoutRef<'a'>> = (args) => {
+  return (
+    <Anchor href='https://www.example.com/' {...args}>
```

Comment:
Maybe better change to `www.bentley.com`.

Reply:
How is that better?

Reply:
Not directing to random URL which might take you anywhere.

Reply:
But it's not a random url. [example.com](https://www.example.com) is an officially reserved URL for documentation/demo purposes